rangeserver/src/server.rs

```diff
@@ -26,23 +27,73 @@ use flatbuf::rangeserver_flatbuffers::range_server::*;
 use proto::rangeserver::range_server_server::{RangeServer, RangeServerServer};
 use proto::rangeserver::{PrefetchRequest, PrefetchResponse};
 
-#[derive(Debug, Default)]
-struct ProtoServer {}
+use crate::prefetching_buffer::PrefetchingBuffer;
+
+pub mod rangeserver {
```

Comment:
why is this needed?

Reply:
I was referring to the rangeserver include, I am able to build despite removing it.

Reply:
Oh I see, you're right.